drs-auth_client/spec/lib/drs/auth_client/client_spec.rb

```diff
@@ -83,6 +83,25 @@
         is_expected.to eql([])
       end
     end
+
+    context "with options" do
+      let(:options) { { key: "value" } }
+      let(:path) { "#{path_prefix}?key=value" }
+      let(:response_hash) { Hash[collection_hash_prefix, [{uid: "UID 7", other: "NAME 7"}]] }
+      let(:response_body) { -> (_) {response_hash.to_json} }
+
+      subject { client.send(collection_resource_method, options) }
+
+      it "makes the correct request with params" do
+        subject
+
+        stubbed_calls.verify_stubbed_calls
+      end
+
+      it "returns only matching #{plural_name}" do
```

Comment:
I'm thinking that this test doesn't have to be here. Because it doesn't test anything else than has already been tested in the scenarios before. I know it seems like we should do it, but it's all stubbed, so maybe we could speed up the suite by removing it.